File: sources/Flush 0.9/me/nyan/flush/utils/other/ChatUtils.java
package me.nyan.flush.utils.other;

import me.nyan.flush.Flush;
import net.minecraft.client.Minecraft;
import net.minecraft.util.ChatComponentText;

public class ChatUtils {
    private static final Minecraft mc = Minecraft.getMinecraft();

    public static void println(Object o) {
        println(String.valueOf(o));
    }

    public static void println(String message) {
        if (mc.thePlayer == null) {
            return;
        }
        mc.thePlayer.addChatMessage(new ChatComponentText(message.isEmpty() ? "" : "§7[§9" + Flush.NAME + "§7] §f" + message));
    }

    public static void println() {
        println("");
    }
}

File: sources/Kevin Reborn/kevin/utils/ChatUtils.kt
/*
 * This program is free software: you can redistribute it and/or modify
 *   it under the terms of the GNU General Public License as published by
 *   the Free Software Foundation, either version 3 of the License, or
 *   (at your option) any later version.
 *
 *   This program is distributed in the hope that it will be useful,
 *   but WITHOUT ANY WARRANTY; without even the implied warranty of
 *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *   GNU General Public License for more details.
 *
 *   You should have received a copy of the GNU General Public License
 *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
package kevin.utils

import kevin.main.KevinClient
import net.minecraft.client.Minecraft
import net.minecraft.util.ChatComponentText

object ChatUtils {
    fun message(message: String){
        Minecraft.getMinecraft().ingameGUI.chatGUI.printChatMessage(ChatComponentText(message))
    }
    fun messageWithStart(message: String){
        Minecraft.getMinecraft().ingameGUI.chatGUI.printChatMessage(ChatComponentText("${KevinClient.cStart} $message"))
    }
}